test/integration/specs/kbaseNarrativeSidePublicTab_data.json

```diff
@@ -0,0 +1,291 @@
+{
+    "cases": {
+        "TEST_CASE_1": {},
+        "TEST_CASE_2": {},
+        "TEST_CASE_3": {},
+        "TEST_CASE_4": {},
+        "TEST_CASE_5": {}
+    },
```

Comment:
is this necessary?

Reply:
This establishes the global test case data, which in this case is absent.
Ideally the test data would be consistent across environments, and most of the test data could live in the global test cases. This can be accomplished for testing certain aspects of test narratives, but creating equivalent narratives. 
For search data, though, unfortunately we load different public data in CI than Prod, which complicates testing, and other areas supporting search.
It might be possible to reconstruct the test data using a search term which produces identical results.